README.md

```diff
@@ -228,7 +226,6 @@ The list below contains the functionality that contributors are planning to deve
 Please refer to the official documentation at [Documentation](https://docs.feast.dev/)
  * [Quickstart](https://docs.feast.dev/getting-started/quickstart)
  * [Tutorials](https://docs.feast.dev/tutorials/tutorials-overview)
- * [Examples](https://github.com/feast-dev/feast/tree/master/examples)
```

Comment:
?

Reply:
It was not added as a part of templates in https://github.com/feast-dev/feast/pull/5194 so `infra/scripts/compile-templates.py` was removing it. Added to the templates now.